src/line.c

```diff
@@ -496,6 +496,15 @@ sline_sline_pos(const SLine *l1, const SLine *l2)
 	vector3d_spoint(&p[2], &v[1][0]);
 	vector3d_spoint(&p[3], &v[1][1]);
 
+	/* check connected lines */
+	if (FPgt(il2->length, 0.0) && (vector3d_eq(&v[0][0], &v[1][0]) ||
+								   vector3d_eq(&v[0][0], &v[1][1]) ||
+								   vector3d_eq(&v[0][1], &v[1][0]) ||
+								   vector3d_eq(&v[0][1], &v[1][1])))
+	{
+		return PGS_LINE_CONNECT;
+	}
+
```

Comment:
Would it be possible to add a test for this change to `sline_sline_pos()`?

Reply:
@esabol I think, the added polygon allows to test this branch in the code. Otherwise, it goes down in the function and returns wrong relationship. The current change just rolls back the old commit. I propose to fix degenerate polygons later, in another commit.

Reply:
OK. Sounds good, @vitcpp.